src/thorlabs_cube/driver/message.py

```diff
@@ -115,6 +115,26 @@ class MGMSG(Enum):
     PZ_SET_TPZ_IOSETTINGS = 0x07D4
     PZ_REQ_TPZ_IOSETTINGS = 0x07D5
     PZ_GET_TPZ_IOSETTINGS = 0x07D6
+    QUAD_SET_PARAMS = 0x0870
+    QUAD_REQ_PARAMS = 0x0871
+    QUAD_GET_PARAMS = 0x0872
+    QUAD_SET_EEPROM_PARAMS = 0x0875
+    QUAD_REQ_STATUSUPDATE = 0x0880
+    QUAD_GET_STATUSUPDATE = 0x0881
+    QUAD_ACK_STATUSUPDATE = 0x0882
+
+
+class SUBMSG(Enum):
```

Comment:
```suggestion
class QUADMSG(Enum):
```
How about this more descriptive name? Makes more clear that these messages are only used on a subset of controllers.